packages/@react-spectrum/actionbar/src/ActionBar.tsx

```diff
@@ -55,7 +55,8 @@ const ActionBarInner = React.forwardRef((props: ActionBarInnerProps, ref: DOMRef
     onAction,
     onClearSelection,
     selectedItemCount,
-    isOpen
+    isOpen,
+    items
```

Comment:
should we pull this out into a separate PR?

Reply:
revisiting this, do we need to update the after_version? or does this need to be pulled out?

Reply:
Updated the after_version. I can pull this out if we need to though. Not sure what the priority of this  component is.